carina-core/src/main/java/com/qaprosoft/carina/core/foundation/filter/IFilter.java

```diff
@@ -63,4 +63,38 @@ default boolean ruleCheck(List<String> ruleExpression, List<String> actualValues
     default boolean ruleCheck(List<String> ruleExpression, String actualValue) {
         return ruleCheck(ruleExpression, Arrays.asList(actualValue));
     }
+
+    default boolean ruleCheckWithoutAnnotation(List<String> ruleExpression) {
```

Comment:
how about default method like:
`default boolean ruleCheck(List<String> ruleExpression)`
so we have the same naming but different number of params